app/code/core/Mage/Log/etc/config.xml

```diff
@@ -29,9 +29,28 @@
             </entities>
         </ignoredModules>
         <ignore_user_agents>
-            <google1>Googlebot/1.0 (googlebot@googlebot.com http://googlebot.com/)</google1>
-            <google2>Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)</google2>
-            <google3>Googlebot/2.1 (+http://www.googlebot.com/bot.html)</google3>
+            <adsbot>AdsBot</adsbot>
+            <alphabot>AlphaBot</alphabot>
+            <amazonbot>Amazonbot</amazonbot>
+            <bingbot>bingbot</bingbot>
+            <blexbot>BLEXBot</blexbot>
+            <dotbot>DotBot</dotbot>
+            <facebookexternalhit>facebookexternalhit</facebookexternalhit>
+            <google>Googlebot</google>
+            <google-site-verification>Google-Site-Verification</google-site-verification>
+            <hrefsbot>AhrefsBot</hrefsbot>
+            <istellabot>istellabot</istellabot>
+            <mauibot>MauiBot</mauibot>
+            <mj12bot>MJ12bot</mj12bot>
+            <mod_pagespeed>mod_pagespeed</mod_pagespeed>
+            <opensiteexplorer>spbot</opensiteexplorer>
+            <pinterestbot>Pinterestbot</pinterestbot>
+            <semrushbot>SemrushBot</semrushbot>
+            <seotester>SEOTesterBot</seotester>
+            <yandexbot>YandexBot</yandexbot>
+            <yandeximage>YandexImages</yandeximage>
+            <yandexmobilebot>YandexMobileBot</yandexmobilebot>
+            <zoominfobot>ZoominfoBot</zoominfobot>
```

Comment:
My opinion is that the list should be as short as possible, limited to the most important bots. Some have nothing to look for there. Everyone can configure their own list.

Reply:
agree with @ADDISON74, although I think this list is ok, in the sense that to have about 20 lines shouldn't make any difference in computation times. but if we were to import like 1000 then it would probably not be a good idea

Reply:
Yes, I leave the decision up to you. For me, the most important thing is the exclusion method.